winter-collector-page.php

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Collected Cats</title>
+    <link href="winter-collector-page.css" rel="stylesheet">
+    <link href="normalize.css" rel="stylesheet">
+</head>
+<body>
+
+<h1>Collected Cats</h1>
+
+<h2>Welcome to your cat collection! Here you can see a list of all your cats!</h2>
+
+
+<?php
+
+$db = new PDO ('mysql:host=db;dbname=winter-collector-page', 'root', 'password');
+
+$query = $db -> prepare("SELECT * FROM cats");
+
+$query -> execute();
+
+$allResults = $query->fetchAll(\PDO::FETCH_ASSOC);
+
+echo "<section>";
+foreach ($allResults as $row) {
+    echo "<div>";
+    echo "<img src='img-id'" . $row["id"] . ".jpg" . "/>";
+    echo "<ul>";
+    echo "<li><h3>Name:</h3> " . $row["name"] . "</li>";
+    echo "<li><h3>Colour:</h3> " . $row["colour"] . "</li>";
+    echo "<li><h3>Weight:</h3> " . $row["weight"] . "kg</li>";
+    echo "<li><h3>Age:</h3> " . $row["age"] . "</li>";
+    echo "<li><h3>Breed:</h3> " . $row["breed"] . "</li>";
+    echo "</ul>";
+    echo "<br>";
+    echo "<br>";
+    echo "<form method='post' action='delete'>";
+    echo "<button type='submit'>delete this cat?</button></form>";
+    echo "</div>";
+}
+echo "</section>";
+
+
+
+?>
+</body>
+
+</html>
+
+
+
+<!--// TABLE?? -->
```

Comment:
If not being used, this should be erased maybe.

Reply:
done. deleted now